src/build-data/oids.txt

```diff
@@ -25,6 +25,13 @@
 1.3.6.1.4.1.2.267.10.3.3 = Kyber-768-90s-r3
 1.3.6.1.4.1.2.267.10.4.4 = Kyber-1024-90s-r3
 
+# Dilithium OIDs are based on the following internet draft
+#  https://www.ietf.org/id/draft-uni-qsckeys-00.html#section-7.1
+#  Dilithium AES modes are not (yet) defined in the list.
```

Comment:
There's a newer revision of that draft which also talks about AES modes. I didn't look deeper into the details though, just pointing it out: https://datatracker.ietf.org/doc/html/draft-uni-qsckeys-01.html#page-25

Reply:
Thanks for the hint @reneme! Although this seems very weird to me: they removed all OIDs from the never version of the draft. The first version had at least some OIDs specified. See this diff here.
![image](https://user-images.githubusercontent.com/44158320/173954788-1e95fd9b-4675-4c88-9e25-b270d0030e07.png)

Reply:
That probably worry about future interoperability, and are afraid to see a zoo of sightly different implementations, each one claiming conformance... So, they wait fur the NIST to (finally!) announce the "chosen" ones.

Reply:
Dilithium is now the chosen one :-) The current draft will expire von "13 November 2022", I hope the next draft to contain the official OIDs.
@reneme I don't want to wait that long, do you have an idea how we can speed this up? I couldn't find any official IODs yet.

Reply:
For the time being we could register the OIDs for Dilithium under [Botan's private OID space](https://botan.randombit.net/handbook/dev_ref/oids.html). Interoperability is a problem, of course, but that won't get fixed before official OIDs have been announced.